TESTING.md

```
@@ -0,0 +1,143 @@
+
+// FORBIDDEN - require()
+jest.mock('../Component', () => {
+  const React = require('react'); // NEVER
+  return () => React.createElement('div');
+});
+
+// FORBIDDEN - JSX in mocks
+jest.mock('../Component', () => () => <div>Mock</div>);
+```
+
+**Mock Custom Hooks with jest.fn()**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)